apps/homework/models.py

```diff
@@ -130,8 +134,9 @@ def retrieve_score_and_status(self):
             data = score_api_resp.json()
             if data.get('status'):
                 self.stored_status = data.get('status')
-                self.stored_score = float(data.get('score', None))
-
+                # .get should already return a default value of None.
```

Comment:
i dont think this comment is necessary, and we shouldn't do `float(None)`:

```
>>> float(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() argument must be a string or a number, not 'NoneType'
```

Reply:
Didn't see this, fixed. (Defaults to 0)

Reply:
cool